### ISSUE TITLE
New issue with GOG games with multiple files per installer?

### ISSUE THREAD

**Issue by btegs** [tkashkin/GameHub#350]
In the past, I would have no issue installing GOG games (both native and the .exe files with Proton) in GameHub like Disco Elysium and River City Girls. For example, River City Girls would have these files and install properly.

setup_river_city_girls_1.1_(35468).exe
setup_river_city_girls_1.1_(35468).exe.md5
setup_river_city_girls_1.1_(35468)-1.bin
setup_river_city_girls_1.1_(35468)-1.bin.md5

Now whenever I download a game it has a download file and a download.md5 no matter the size of the file. I tried downloading a game with a single file today (Flashback) and the single file installer worked fine and I was able to boot it. Flashback was a bad example as it currently has a borked status on ProtonDB but at least I was able to start the GOG installer.

I did try a native Linux game (Blade Runner) that was only one file and the game would get stuck on installing and just sit there. It did create a Blade Runner installation directory, but nothing went into it.

But when I tried a game with two parts (XIII on a sale), it only had the single download file even if the game is two parts in the offline installers section of GOG. Tried InnoExtract, Proton, and WineWrap.

Is this a new issue or something that changed in having the download just be "download" instead of downloading the file as-is from the servers? I'm curious as I have noticed this recently.

**Comment by tkashkin** [comment]
I don't think I've changed anything related recently. Maybe something has changed on GOG side.

Try to run GameHub in terminal with `--debug --verbose --log-downloader` options and post a log here.

**Comment by btegs** [comment]
Here's me starting the download of Disco Elysium, which has these files listed in the account page:

Disco Elysium (Part 1 of 4) 579003ab 1 MB
Disco Elysium (Part 2 of 4) 579003ab 4 GB
Disco Elysium (Part 3 of 4) 579003ab 4 GB
Disco Elysium (Part 4 of 4) 579003ab 3.7 GB
Patch (5bad0c96 to 579003ab) 579003ab 49 MB
Patch (4801e0fb to bbe2afa0) bbe2afa0 6 MB
Patch (1be313bc to 4801e0fb) 4801e0fb 49 MB
Patch (a54d931a to 1be313bc) 1be313bc 66 MB

```
brian@pop-os:~$ gamehub --debug --verbose --log-downloader
[DEBUG]  [Database.migrate] Latest db version: 11, current: 11
[DEBUG]  [WineWrap] Updating index
[DEBUG]  [DOSBox.init] Config directory: '/usr/share/com.github.tkashkin.gamehub/compat/dosbox'
[DEBUG]  [DOSBox.init] Config: '/usr/share/com.github.tkashkin.gamehub/compat/dosbox/windowed.conf'; description: 'Disable fullscreen'; enabled: true
[DEBUG]  [DOSBox.init] Config directory: '/home/brian/.local/share/com.github.tkashkin.gamehub/compat/dosbox'
[DEBUG]  [DOSBox.init] Config directory: '/home/brian/.config/com.github.tkashkin.gamehub/compat/dosbox'
[INFO]   - GameHub
[INFO]       Version: 0.15.0-2c73d30-dev
[INFO]       Branch:  dev
[INFO]       Commit:  2c73d3096b914708789ee9f0ff95d6c7c4d97d3e
[INFO]   - Environment
[INFO]       Distro:  Pop!_OS 19.10
[INFO]       DE:      pop:GNOME
[INFO]       GTK:     3.24.12
[INFO]       Themes:  Pop | Pop
[DEBUG]  [WineWrap] Index updated
[DEBUG]  [SteamGame] 403640: no cached app data for 'Dishonored 2', fetching...
[DEBUG]  [GOG] Loading games: page 1 of 3
[DEBUG]  [GOG] Imported tag: COMPLETED (182789384)
[DEBUG]  [GOG] Imported tag: NEXT TO PLAY (134948834)
[DEBUG]  [GOG] Imported tag: BACKLOG (87108284)
[DEBUG]  [GOG] Imported tag: FAVORITE (39267734)
[DEBUG]  [SoupDownloader] Downloading 'http://cdn.akamai.steamstatic.com/steam/apps/403640/header.jpg'...
[DEBUG]  [GOG] Loading games: page 2 of 3
[DEBUG]  [SoupDownloader] Content-Length: 22747
[DEBUG]  [SoupDownloader] Downloaded 'http://cdn.akamai.steamstatic.com/steam/apps/403640/header.jpg'
[DEBUG]  [GOG] Loading games: page 3 of 3
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/90fbf2c762cc6d39ad80ba64b1b065056da9dfc4.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/517e4499671efaf952d960c28e9cf96dce3122a0.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/968b8b6bdcfc5302ef7b9a31f7243634ed8e4123.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/d0b6b890d0d4f4bcaef198af7d3f7d70209b4bd9.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/9b82b80de1286264936f8ff45ef8e98c738aea70.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a00ee030fb05dc32332fd17ed7a88cc8acbda551.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/4da176d8c8a2195aac7d1e0c81084a3f4a8def8e.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/67688ed9b13e632c3032600603e770c22060f12a.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/3f737ea2ef586f2583ddfe18d3856299460a2ef9.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/8ea89bf9e961c74d4ec8cbdca9ea16afd3ee94d6.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/007a5156866dc359b47ed672b9a17f370b901d68.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/fdedf389b3059c369efb2a57e83cc6e402fe1a8d.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a469e4d9c3b8b3fed6d1f39026c8a4fbc49ad39a.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/84bc447e4ddb0d55d281036912de3632f7d9434e.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/09399590d6bbfe94a7afe2f4f788a5e7d72cf449.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ff1eb761fdae0be86ba52b55177b375bd8cdd8b8.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/86de01d3679dbc2f65b5b76fb495a2b43afbb46e.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/07e0cb9c6e8fb4f28ad88310ae5753ce69906f91.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/0b9517c9f669c47975ea43d3c29a673fa0a6d830.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ab8f09e0381902cc1a7415a4cd3b3c20b5b93f11.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/dc22f69c807a5f0f3aa69d057581944c71f84a4d.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/160eb31dae31c85d8158d0b38e8de8b0ab5594bc.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/4e90f556b254a1c42d1d85eff72e48e3d76c1c68.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/306cc9f5cf4d8d1407e8f566bcd6c0f0c950fa1f.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/5722d77d86496652f2cd9497ca292218f876df6b.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/34af7c6659ef56d4898eda8ec1b84023154f5073.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/acf8f5deb3b89f991ba6b52db30e120e60deb3f9.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/245ff89191a9f1d43a5825c3a9fd808259852047.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/613a98039fa6417d22c41a2b2ca8f47d058846c5.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/27e9ed66292c476efa498d5772c56e865d3ba87e.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/c2982b2122a192733c0e7e8d460dc183f3e48903.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/2feb5fcabd025298463d6f9cd0c2e9b22b9ab22d.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/53945a504d6d335719940339de3052aaa19372f0.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/749c83b43f4e7646a2b34bd4f5b58eb982b361f8.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/eb8fa579771016ef20080ea5eaadde5c22fd40a9.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ff38dc75ad6e128e65a3a44be3aed7f1f77ba534.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/60b6355c7469dade22129f62e4c4dfeb56ee4369.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/c4dbe120e9f0f352e8d9d82c011b2f52bf32ed81.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ca276972922c17e6844d4860a9645aeb2272d253.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/69e94d73b2748aacf2a2062dd1138f932a415e4d.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/8673d6744dba6a0d475222d42af9c3959261efef.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ce48cc67d98a8a34b330a09ec5e07920db656c46.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/fe49a9c5654c779094f77557ab95d5401a26ccde.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a404d798e97167660df3fa78a8b63ef7cc3b89a2.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/04504172910c8f2390de21235e22b2d8fe4b536d.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/3fd23c91c575b4725b138f603977979621bcebb5.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/b89eb861690b541a66c6e325ea194eaf56f0f411.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/1c1de8caa36faa62013b4ad9ee5e3c82f58d0f4b.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ce4e8759bd2637bf3d9ecfc0828337b2f492dbd7.jpg'...
[DEBUG]  [SoupDownloader] Downloading 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/49c872652d36802f122390b1db6d5abd34c11e94.jpg'...
[DEBUG]  [SoupDownloader] Content-Length: 2198
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/d0b6b890d0d4f4bcaef198af7d3f7d70209b4bd9.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1838
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a00ee030fb05dc32332fd17ed7a88cc8acbda551.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2269
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/3f737ea2ef586f2583ddfe18d3856299460a2ef9.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2019
[DEBUG]  [SoupDownloader] Content-Length: 1945
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/8ea89bf9e961c74d4ec8cbdca9ea16afd3ee94d6.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/4da176d8c8a2195aac7d1e0c81084a3f4a8def8e.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1732
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/9b82b80de1286264936f8ff45ef8e98c738aea70.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1925
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/86de01d3679dbc2f65b5b76fb495a2b43afbb46e.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1792
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/517e4499671efaf952d960c28e9cf96dce3122a0.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2152
[DEBUG]  [SoupDownloader] Content-Length: 2268
[DEBUG]  [SoupDownloader] Content-Length: 2194
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/90fbf2c762cc6d39ad80ba64b1b065056da9dfc4.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/07e0cb9c6e8fb4f28ad88310ae5753ce69906f91.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/0b9517c9f669c47975ea43d3c29a673fa0a6d830.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1878
[DEBUG]  [SoupDownloader] Content-Length: 1758
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/968b8b6bdcfc5302ef7b9a31f7243634ed8e4123.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/67688ed9b13e632c3032600603e770c22060f12a.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2324
[DEBUG]  [SoupDownloader] Content-Length: 1974
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ab8f09e0381902cc1a7415a4cd3b3c20b5b93f11.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/dc22f69c807a5f0f3aa69d057581944c71f84a4d.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2075
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/160eb31dae31c85d8158d0b38e8de8b0ab5594bc.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2210
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/4e90f556b254a1c42d1d85eff72e48e3d76c1c68.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2215
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/306cc9f5cf4d8d1407e8f566bcd6c0f0c950fa1f.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1939
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/09399590d6bbfe94a7afe2f4f788a5e7d72cf449.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2139
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ff1eb761fdae0be86ba52b55177b375bd8cdd8b8.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1739
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/fdedf389b3059c369efb2a57e83cc6e402fe1a8d.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2164
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/acf8f5deb3b89f991ba6b52db30e120e60deb3f9.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1938
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/5722d77d86496652f2cd9497ca292218f876df6b.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1733
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/34af7c6659ef56d4898eda8ec1b84023154f5073.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1961
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/007a5156866dc359b47ed672b9a17f370b901d68.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1953
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/245ff89191a9f1d43a5825c3a9fd808259852047.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1988
[DEBUG]  [SoupDownloader] Content-Length: 1913
[DEBUG]  [SoupDownloader] Content-Length: 2232
[DEBUG]  [SoupDownloader] Content-Length: 2049
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/84bc447e4ddb0d55d281036912de3632f7d9434e.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/613a98039fa6417d22c41a2b2ca8f47d058846c5.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/27e9ed66292c476efa498d5772c56e865d3ba87e.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/c2982b2122a192733c0e7e8d460dc183f3e48903.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1921
[DEBUG]  [SoupDownloader] Content-Length: 2083
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/2feb5fcabd025298463d6f9cd0c2e9b22b9ab22d.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/53945a504d6d335719940339de3052aaa19372f0.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2112
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/749c83b43f4e7646a2b34bd4f5b58eb982b361f8.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1920
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/eb8fa579771016ef20080ea5eaadde5c22fd40a9.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2254
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ff38dc75ad6e128e65a3a44be3aed7f1f77ba534.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2262
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/60b6355c7469dade22129f62e4c4dfeb56ee4369.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2335
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/c4dbe120e9f0f352e8d9d82c011b2f52bf32ed81.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2327
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/69e94d73b2748aacf2a2062dd1138f932a415e4d.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1651
[DEBUG]  [SoupDownloader] Content-Length: 2112
[DEBUG]  [SoupDownloader] Content-Length: 2076
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ca276972922c17e6844d4860a9645aeb2272d253.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/8673d6744dba6a0d475222d42af9c3959261efef.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ce48cc67d98a8a34b330a09ec5e07920db656c46.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1741
[DEBUG]  [SoupDownloader] Content-Length: 2335
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/fe49a9c5654c779094f77557ab95d5401a26ccde.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/3fd23c91c575b4725b138f603977979621bcebb5.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 1986
[DEBUG]  [SoupDownloader] Content-Length: 1691
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a404d798e97167660df3fa78a8b63ef7cc3b89a2.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/1c1de8caa36faa62013b4ad9ee5e3c82f58d0f4b.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2322
[DEBUG]  [SoupDownloader] Content-Length: 2043
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/04504172910c8f2390de21235e22b2d8fe4b536d.jpg'
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/b89eb861690b541a66c6e325ea194eaf56f0f411.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2016
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/ce4e8759bd2637bf3d9ecfc0828337b2f492dbd7.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2143
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/a469e4d9c3b8b3fed6d1f39026c8a4fbc49ad39a.jpg'
[DEBUG]  [SoupDownloader] Content-Length: 2206
[DEBUG]  [SoupDownloader] Downloaded 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps/403640/49c872652d36802f122390b1db6d5abd34c11e94.jpg'
[DEBUG]  [SoupDownloader] Downloading 'https://content-system.gog.com/products/1771589310/users/47443356004024962/namespaces/website/download?path=%2Fsecure%2Foffline%2F1771589310%2F1771589310%2F53082366807525513%2F4763%2Fsetup_disco_elysium_579003ab_%2864bit%29_%2836214%29-3.bin&token=eyJhbOigciJIUzI1NiIsImV4cCI6IjE1ODM0OTc2MjkifQ.eyJjb250ZW50Ijoie1widXNlcl9pZFwiOjQ3NDQzMzU2MDA0MDI0OTYyLFwicHJvZHVjdF9pZFwiOjE3NzE1ODkzMTAsXCJwYXRoXCI6XCJcXFwvc2VjdXJlXFxcL29mZpxmVcbpXFwvMTc3MTU4OTMxMFxcXC8xNzcxNTg5MzEwXFxcLzUzMDgyMzY2ODA3NTI1NTEzXFxcLzQ3NjNcXFwvc2V0dXBfZGlzY29fZWx5c2l1bV81NzkwMDNhYl8oNjRiaXQpXygzNjIxNCktMy5iaW5cIn0iLCJleHAiOiIxNTgzNDk3NjI5In0.tyDq5Ka_NQHFBEQX5LoYbhWs3NdAB_Nm2SxdGecfJWI&access_token=GAHv90275a46x5tEz6WGn04FIMQrRDUIeuhoWxqrjGFxYlX24w6NNoijAv5DaVw2K0rPMPfx2iw79qfvuXDfumxjyhqMdLsirJVznEMncLlqKu8Z1xY9scvqw0J1dqK4Tu9JekJvWydWdhsPeLK7Zg'...
[DEBUG]  [SoupDownloader] Content-Length: 4296
[DEBUG]  [SoupDownloader] Downloading to '/home/brian/Games/_Collection/GOG/Disco Elysium/download'
[DEBUG]  [SoupDownloader] Content-Length: 3976177421
[DEBUG]  [SoupDownloader] Downloading to '/home/brian/Games/_Collection/GOG/Disco Elysium/download'
```

**Comment by tkashkin** [comment]
It seems that GOG has changed download URLs (not a problem) and stopped returning `Content-Disposition` header (it should contain filename). Now filename needs to be extracted from URL.

**Comment by tkashkin** [comment]
Should be fixed in 1275a10.

**Comment by btegs** [comment]
Seems good so far!
